handlers/controller.js

```diff
@@ -157,6 +122,68 @@ let addProductByLookup = async(req, res) => {
     });
 };
 
+
+/**
+ * Returns the top five manufacturers of the same category of product
+ * @param req request 
+ * @param res  response
+ */
+let getTopManufacturers = async(req, res) => {
+    connectDb().then(async () => {
+        if(!req.params.id) {
+            return res.status(StatusCode.PRECONDITION_FAILED).send(null);
```

Comment:
A `null` value doesn't need to be sent here.

```suggestion
return res.status(StatusCode.PRECONDITION_FAILED).send();
```